keyboards/splitkb/zima/keymaps/default/readme.md

```diff
@@ -0,0 +1,106 @@
+# The default keymap for zima
+
+This includes support for the OLED and Encoder.  However, the actual code is found in the `zima.c` file. This can be replaced by adding your own `oled_task_user(void)` and `encoder_update_user` functinons. These will replace what is in the keyboard, and allow you to customize these features. 
+
+The reason that this is done this way, is so that this functionality will work on the [QMK Configurator](https://config.qmk.fm/#/splitkb/zima/LAYOUT_ortho_4x3)
+
+For reference, the code that is being used: 
```

Comment:
Is there any value in adding this here?
The location of the code is mentioned in the text and if the code changes this will be outdated and confusing.

Reply:
Wanted to make it easily accessible, without having to link to the keyboard's file.